tests/testthat/test-lm-lin.R

```diff
@@ -84,7 +84,7 @@ test_that("Test LM Lin", {
   )
 
   expect_equivalent(
-    lm_lin_out$estimate,
+    lm_lin_out$coefficients,
```

Comment:
In your tests, use `coef(lm_lin_out)` instead of `lim_lin_out$coefficients`

Reply:
oh boy